Release `oas` cli as part of installing optic

JaapRood opened the pull request:
A quiet little soft launch of the CLI we've been building for authoring OpenAPI specs: `oas`. For now, a separate command from the rest of `@useoptic/optic` and the `optic` command, but bundled and installed as part of it.

`@useoptic/openapi-cli` is kept around for now as the home for the code, as well as library use by some of our design partners.

### QA
- [x] run `optic` and `oas` from installing `@useoptic/openapi-cli@0.27.0-1`

niclim commented:
I get an error with 
```
➜  ~ npm i -g @useoptic/optic@0.27.0-0

added 322 packages, removed 4 packages, changed 218 packages, and audited 541 packages in 21s

98 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
➜  ~ oas
/Users/nicholaslim/.nvm/versions/node/v16.14.0/bin/oas: line 1: use strict: command not found
/Users/nicholaslim/.nvm/versions/node/v16.14.0/bin/oas: line 3: syntax error near unexpected token `('
/Users/nicholaslim/.nvm/versions/node/v16.14.0/bin/oas: line 3: `var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");'

```

JaapRood commented:
> I get an error with
> 
> ```
> ➜  ~ npm i -g @useoptic/optic@0.27.0-0
> 
> added 322 packages, removed 4 packages, changed 218 packages, and audited 541 packages in 21s
> 
> 98 packages are looking for funding
>   run `npm fund` for details
> 
> found 0 vulnerabilities
> ➜  ~ oas
> /Users/nicholaslim/.nvm/versions/node/v16.14.0/bin/oas: line 1: use strict: command not found
> /Users/nicholaslim/.nvm/versions/node/v16.14.0/bin/oas: line 3: syntax error near unexpected token `('
> /Users/nicholaslim/.nvm/versions/node/v16.14.0/bin/oas: line 3: `var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");'
> ```

Yup, thank you prereleases! Should be fixed through https://github.com/opticdev/optic/pull/1296/commits/6a672f4b552c333ea26b2f2dd69a969e687ad65c, releasing as part of 0.27.0-1 as we speak.
